app/src/main/java/com/example/firebaseloginapp/AccountActivity/ResetPasswordActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.example.firebaseloginapp.AccountActivity;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ProgressBar;
+import android.widget.Toast;
+
+import com.example.firebaseloginapp.R;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class ResetPasswordActivity extends AppCompatActivity {
+
```

Comment:
Blank line. In the LoginActivity you don't have blank line before class members declaration.